js/client/modules/@arangodb/testutils/crash-utils.js

```diff
@@ -596,7 +596,7 @@ function generateCrashDump (binary, instanceInfo, options, checkStr) {
 function aggregateDebugger(instanceInfo, options) {
   print("collecting debugger info for: " + JSON.stringify(instanceInfo.getStructure()));
   if (!instanceInfo.hasOwnProperty('debuggerInfo')) {
-    print("No debugger info persisted to " + instanceInfo.debuggerInfo.getStructure());
+    print("No debugger info persisted to " + JSON.stringify(instanceInfo.getStructure()));
```

Comment:
Already have above
```suggestion
    print("No debugger info persisted");
```

Reply:
no, we want the dump of the instance here.

Reply:
We dump it above!